nemo/collections/asr/parts/utils/streaming_utils.py

```diff
@@ -343,6 +343,129 @@ def inplace_buffer_merge(buffer, data, timesteps, model):
     return buffer
 
 
+class StreamingFeatureBufferer:
+    """
+    Class to append each feature frame to a buffer and return an array of buffers.
+    This class is designed to perform a real-life streaming decoding where only a single chunk
+    is provided at each step of a streaming pipeline.
+    """
+
+    def __init__(self, asr_model, chunk_size=1.6, buffer_size=5.0, chunk_look_back=0.01):
```

Comment:
Maybe not having default value for chunk_size and buffer_size as user need to specify them? Your call.

Reply:
removed default values